articles/140_resource_names.md

```diff
@@ -19,13 +19,33 @@ published: true
 
 Original Author: {{ page.author }}
 
-## Context
+## Background
 
-Before proposing constraints for ROS 2 topic and service names and a mapping to underlying DDS topics, this article will first summarize the existing guidelines and restrictions for both ROS 1 and DDS.
+### Concepts
 
-### ROS 1 Names
+#### Content-oriented resources
```

Comment:
> I think it would be good to motivate our use of this terminology

I agree, and I think that CCN (Content Centric Networking) gets pretty close. A related [Linux Foundation project](https://wiki.fd.io/view/Cicn) adds Publish/Subscribe Networking as a potential alias for the same concept. See 7680627ee8f2fa8675fd58e72f814585c6721f2a.

> The URI page talks about host and path, and that host is optional:

For URIs, a host is always defined, either explicitly or implicitly through a sane default e.g. `localhost` for `file://` URIs, see [RFC 3986 Section 3.2.2](https://tools.ietf.org/html/rfc3986#section-3.2.2).